lib/cli.js

```diff
@@ -12,9 +12,14 @@ var meow = require('meow');
 var list = require('cli-list');
 var pkg = require('../package.json');
 var Router = require('./router');
-
 var gens = list(process.argv.slice(2));
 
+/* eslint new-cap: 0, no-extra-parens: 0 */
+var tabtab = new (require('tabtab').Commands.default)({
```

Comment:
This `.default` thing really annoys me, but it's due to not being able to `import` it from yeoman. Or I'd need to change tabtab to not use ES6 modules.